overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
```

Comment:
This indentation makes it look like we didn't run gofmt on the code; is there any chance we could gofmt all of the changes to make sure the formatting is consistent across the board?

Reply:
I ended up running gofmt on all 3 .go files I've modified in https://github.com/ameihm0912/geomodel/pull/10/commits/cdbff7e4fed3dc12537efd5ad81054151ab39019